packages/lesswrong/components/community/modules/CommunityMembers.tsx

```diff
@@ -259,7 +253,9 @@ const CommunityMembers = ({currentUser, userLocation, distanceUnit='km', locatio
             <div className={classes.location}>{hit.mapLocationAddress}</div>
           </div>
         </div>
-        {hit.htmlBio && <div className={classes.description}><div dangerouslySetInnerHTML={{__html: hit.htmlBio}} /></div>}
+        {hit.htmlBio && <ContentStyles contentType="comment" className={classes.description}>
+          <div dangerouslySetInnerHTML={{__html: truncatise(hit.htmlBio, {TruncateBy: 'characters', TruncateLength: 160})}} />
+        </ContentStyles>}
```

Comment:
You might consider using truncate from our code's wrapper instead of directly from the library?